### ISSUE TITLE
Process out of memory

### ISSUE THREAD

**Issue by andyphilipp** [jspm/jspm-cli#868]
```shell
=> jspm install material-design-icons
     Updating registry cache...
     Looking up github:google/material-design-icons
     Downloading github:google/material-design-icons@2.0.0
FATAL ERROR: CALL_AND_RETRY_LAST Allocation failed - process out of memory
Abort trap: 6
```

**Comment by guybedford** [comment]
Try running `jspm config registries.github.maxRepoSize 1000` and see if that helps?

**Comment by andyphilipp** [comment]
Running ```jspm config registries.github.maxRepoSize 1000``` didn't seem to solve the issue, I'm still getting the same error.

**Comment by guybedford** [comment]
@andyphilipp perhaps follow some of the suggestions in a post like http://stackoverflow.com/questions/29442965/fatal-error-js-allocation-failed-process-out-of-memory-aborted-core-dumped.

**Comment by starx** [comment]
@guybedford I am also facing this issue for about a week now. Tried many solution didn't work. This has to be a jspm issue.

**Comment by guybedford** [comment]
@starx did you try the suggestion in http://stackoverflow.com/a/26094800/2305274 here for increasing the memory size in Node?

**Comment by adamnfish** [comment]
In case it's useful, you can see the memory usage running away on a `jspm install` command here. Allowing more memory might fix the problem but the fact that it is using so much in the first place seems like the problem here?

After the gif finished recording it hung for a bit and then died with a memory error.

```
FATAL ERROR: CALL_AND_RETRY_LAST Allocation failed - process out of memory
Aborted (core dumped)
```

![gifrecord_2015-08-17_180027](https://cloud.githubusercontent.com/assets/29761/9310803/1098eb26-450a-11e5-9297-e66ebb7e1bf0.gif)

**Comment by theefer** [comment]
I've been able to reproduce this issue with Node 0.10 and 0.12, jspm 0.15 and 0.16-beta.7. Seems to be some issue specific to that `material-design-icons` package.

Does anyone have guidance to inspect the node process and see if it's hanging somewhere specific in JSPM?

**Comment by guybedford** [comment]
Thanks I've posted an issue at https://github.com/bower/decompress-zip/issues/48.

**Comment by guybedford** [comment]
This should be fixed in https://github.com/jspm/github/commit/e45072e40cfd505ae6d8a746904344e2d082523e. I've released a 0.13.2 of SystemJS builder which will apply with an npm update. Let me know if you have any further issues at all.
